src/hooks/useAccounts.tsx

```
@@ -47,46 +54,28 @@ const useAccountsContext = () => {
     disconnect: disconnectFuel,
   } = useFuelWallet();
 
+  const isPrivyConnected = Boolean(authenticated && user);
+  const connectedWalletAddress = ready && isPrivyConnected ? user?.wallet?.address : fuelAddress;
   const hasSubAccount = useAppSelector(BonsaiCore.account.parentSubaccountSummary.data) != null;
```

ESLint

[error] 'hasSubAccount' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)